modules/recon/contacts-credentials/hibp_paste.py

```diff
@@ -49,15 +52,19 @@ def module_run(self, accounts):
                     if paste['Source'] in sites:
                         fileurl = sites[paste['Source']].format(paste['Id'])
                         download = self.options['download']
-                    elif self.options['download'] == True:
+                    elif self.options['download']:
                         self.alert(f"Download not available for {paste['Source']} pastes.")
                     self.alert(f"{account} => Paste found! Seen in a {paste['Source']} on {paste['Date']} ({fileurl}).")
-                    if download == True:
-                        resp = self.request(fileurl)
+                    if download:
+                        try:
+                            resp = self.request(fileurl)
+                        except ConnectionError:
+                            self.alert(f"Paste could not be downloaded ({fileurl}).")
+
                         if resp.status_code == 200:
```

Comment:
What does a ConnectionError include? If there is not a resp object at this point due to a timeout, etc. then there is going to be an unhandled error at this point because a resp object won't exist. Did you test this condition?